addons/l10n_br_website_sale/models/website.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for website in vals_list:
+            if self.env['res.company'].browse(website['company_id']).country_code == "BR":
+                website.update({
+                    'show_line_subtotals_tax_selection': 'tax_included',
+                })
```

Comment:
Oopsi my bad, I totally fucked up here
Better way to do it:
```suggestion
                website['show_line_subtotals_tax_selection'] = 'tax_included'
```